rdr_service/dao/consent_dao.py

```diff
@@ -13,65 +14,73 @@ class ConsentDao(BaseDao):
     def __init__(self):
         super(ConsentDao, self).__init__(ConsentFile)
 
-    def get_participants_with_consents_in_range(self, start_date, end_date=None) -> List[ParticipantSummary]:
-        with self.session() as session:
-            query = session.query(
-                ParticipantSummary
-            ).join(
-                Participant,
-                Participant.participantId == ParticipantSummary.participantId
-            ).filter(
-                ParticipantSummary.participantOrigin == 'vibrent',
-                Participant.isGhostId.isnot(True),
-                Participant.isTestParticipant.isnot(True),
+    @classmethod
+    def get_participants_with_consents_in_range(cls, session, start_date,
+                                                end_date=None) -> Collection[ParticipantSummary]:
+        query = session.query(
+            ParticipantSummary
+        ).join(
+            Participant,
+            Participant.participantId == ParticipantSummary.participantId
+        ).filter(
+            ParticipantSummary.participantOrigin == 'vibrent',
+            Participant.isGhostId.isnot(True),
+            Participant.isTestParticipant.isnot(True),
+            or_(
+                ParticipantSummary.email.is_(None),
+                ParticipantSummary.email.notlike('%@example.com')
```

Comment:
I think I had specifically left the Test HPO out just in case they were real participants that had been inadvertently paired to Test. But the same could happen for ghost or test too. I think it would be good to revisit this part when discovering participants that need consents copied for reasons other than authored date (such as changing orgs) if that's ok with you.